lib/Experian/IDAuth.pm

```diff
@@ -387,6 +387,8 @@ sub _get_result_proveid {
     my $kyc_dob = $kyc_summary->findvalue('DateOfBirth/Count') || 0;
     my $cr_total = $credit_reference->findvalue('TotalNumberOfVerifications')
       || 0;
+    $decision->{num_verifications} = $cr_total;
```

Comment:
If we have the requirement to provide the field, we need a test case for it. And we need documentation.